src/Data/PlaceCal/Events.elm

```diff
@@ -46,8 +46,11 @@ type alias EventLocation =
 
 
 type alias Geo =
-    { latitude : String
-    , longitude : String
+    -- Bug: We expect if there is a postcode in the address, these exist.
+    -- But, in practice, sometimes they don't see:
+    -- https://github.com/geeksforsocialchange/PlaceCal/issues/1639
+    { latitude : Maybe String
```

Comment:
Note I am not changing these property keys to `maybeLatitude` and `maybeLongitude` because I expect this bug to be temporary and that seemed to obscure for me.

Reply:
i think the problem is on PC side: it really should return null on the geo field if the data is missing. how likely is there to be a latitude without a longitude?

Reply:
It is an error in PlaceCal - but Kim and I decided there is no harm guarding against it here.
https://github.com/geeksforsocialchange/PlaceCal/issues/1639

It is unlikely to have lat without long or vice-versa but nothing is impossible as we know so it feels less arbitrary to do it definitively this way.